packages/solidstart/src/middleware.ts

```diff
@@ -8,19 +8,13 @@ export type ResponseMiddlewareResponse = Parameters<ResponseMiddleware>[1] & {
 };
 
 function addMetaTagToHead(html: string): string {
-  const { 'sentry-trace': sentryTrace, baggage } = getTraceData();
+  const metaTags = getTraceMetaTags();
 
-  if (!sentryTrace) {
+  if (!metaTags) {
     return html;
   }
 
-  const metaTags = [`<meta name="sentry-trace" content="${sentryTrace}">`];
-
-  if (baggage) {
-    metaTags.push(`<meta name="baggage" content="${baggage}">`);
-  }
-
-  const content = `<head>\n${metaTags.join('\n')}\n`;
+  const content = `<head>\n${metaTags}\n`;
   return html.replace('<head>', content);
```

Comment:
Just wondering what happens if a user has any attributes set on the `<head>`tag like `lang="de"` or something like this? 🤔

Reply:
Yea then no meta-tags, but I have never seen attributes placed on head ever before. Are there any such cases? (also our other sdks will have this problem too).

Reply:
I think it's technically possible although very uncommon